examples/transactions/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "examples-transactions"
+
+publish.workspace = true
+version.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+authors.workspace = true
+license.workspace = true
+homepage.workspace = true
+repository.workspace = true
+
+[dev-dependencies]
+alloy-network.workspace = true
+alloy-node-bindings.workspace = true
+alloy-primitives.workspace = true
+alloy-provider.workspace = true
+alloy-rpc-types.workspace = true
+alloy-rpc-client.workspace = true
+alloy-rpc-trace-types.workspace = true
+alloy-sol-types = { workspace = true, features = ["json"] }
+alloy-transport.workspace = true
+alloy-transport-http.workspace = true
```

Comment:
Please use the new Alloy namespace (see: https://github.com/alloy-rs/examples/pull/11)